src/main/java/com/sonyericsson/jenkins/plugins/bfa/PluginImpl.java

```diff
@@ -437,6 +462,20 @@ public void setNrOfScanThreads(int nrOfScanThreads) {
     }
 
     /**
+     * Checks if the build with certain result should be analyzed or not.
+     *
+     * @param result the result
+     * @return true if it should be analyzed.
+     */
+    public static boolean needToAnalyze(Result result)  {
+        if (getInstance().doNotAnalyzeAbortedJob()) {
```

Comment:
OK, if you want to add it in the future you can look at the class name of the cause to avoid adding the dependency. e.g.
```java
if (cause.getClass().getName().equals("org.jenkinsci.plugins.WhateverTheNameNowIs"))
```
A bit ugly but should work ;)